src/keylessBackup/types.ts

```diff
@@ -6,7 +6,8 @@ export enum KeylessBackupFlow {
 export enum KeylessBackupStatus {
   NotStarted = 'NotStarted',
   InProgress = 'InProgress',
-  RestoreZeroBalance = 'RestoreZeroBalance',
+  RestoreZeroBalance = 'RestoreZeroBalance', // only in restore flow
   Completed = 'Completed',
   Failed = 'Failed',
+  NotFound = 'NotFound', // only in restore flow
```

Comment:
Maybe this is just me, but I find mixing the states between recovery and backup into a single enum (and piece of redux state) a little confusing. I understand why we do this, but at first glance it is a bit confusing to me exactly what each state represents between the different flows - the semantics of each state are different depending on the context. It's also not immediately clear if _all_ states are possible in recovery, and only a subset in setup (that is indeed the case, but again, not obvious on first glance). This is all pretty easy to grok though once you go through the code so nothing to really do here I think, just wanted to bring it up.

Reply:
I'm with you that I think this is a stretch to have them combined jointly. I cringed a little adding this `NotFound` but didn't think it was worth refactoring everything to be separate between restore and setup at this point. Thanks for brining it up for discussion!